.github/workflows/test.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           filename: ../../stats.txt
+        if: github.event_name == 'pull_request'
```

Comment:
do we still need this, we are having this here right: https://github.com/cosmos/qa-integration/pull/35/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R5

Reply:
Example: https://github.com/cosmos/qa-integration/runs/6934837226?check_suite_focus=true

Reply:
I guess we can remove master from `on` setting to avoid that

Reply:
Not only with master, but when we try to run actions manually on specific branch using `workflow_dispatch`, this job will fail. So, I thought it would be better to have some check.